packages/autorest.typescript/src/utils/valueHelpers.ts

```diff
@@ -83,6 +84,21 @@ export function getStringForValue(
   }
 }
 
+function getStringForSequence(
+  value: any,
+  valueType: AllSchemaTypes | MapperTypes | string
+) {
+  if (Array.isArray(value)) {
+    const valuesArr = [];
+    for (let item of value) {
+      valuesArr.push(getStringForValue(item, valueType));
+    }
+    return `[${valuesArr.join(", ")}]`;
+  }
+  const valueStr = getStringForValue(value, valueType);
+  return `[${valueStr}]`;
+}
```

Comment:
Can you consider the way we build value for samples ?

Reply:
Yes we handle this in a similar way in samples but the input is different because the sample we use Example as input.